packages/beacon-node/src/chain/archiveStore/utils/xdelta3.ts

```diff
@@ -0,0 +1,20 @@
+import {decodeSync, encodeSync} from "@chainsafe/xdelta3-node";
+import {IStateDiffCodec} from "../interface.js";
+
+export class XDelta3Codec implements IStateDiffCodec {
```

Comment:
```suggestion
export class BinaryDiffCodec implements IStateDiffCodec {
```
don’t think we need to expose the name xdelta3 in our code base. Unlikely we will adopt another binary codec in the future in lodestar